### PR TITLE
New chyron color fields in projector

### LINKED ISSUE #2437: Need a new color setting for the projector
Needed by https://github.com/OpenSlides/openslides-client/issues/3655

It should be a setting like `chyron_background_color` and should be in the projector model and needs update in the actions. E.g. `chyron_agenda_item_background_color` or `chyron_background_color_2` or so, it should be called.

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Closes #2437 
Needs OpenSlides/openslides-meta#104

### CODE REVIEW

File: openslides_backend/migrations/migrations/0053_fill_new_color_fields.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+from datastore.migrations import BaseModelMigration
+from datastore.shared.util import fqid_from_collection_and_id
+from datastore.writer.core import BaseRequestEvent, RequestUpdateEvent
+
+from ...shared.filters import FilterOperator
+
+
+class Migration(BaseModelMigration):
+    """
+    This migration removes all remnants of the old structure level field in meeting users.
```
Comment (reiterl) [inline openslides_backend/migrations/migrations/0053_fill_new_color_fields.py:10]: Change the comment

**Review by reiterl — COMMENTED** [review]
One comment needs improvement.

File: openslides_backend/migrations/migrations/0053_fill_new_color_fields.py (hunk 1 above)
Comment (luisa-beerboom) [reply openslides_backend/migrations/migrations/0053_fill_new_color_fields.py]: Done

**Review by reiterl — APPROVED** [review]
Looks okay now.

**Pull request merged**